src/frontend/screens/Login/components/SIDLogin/index.tsx

```diff
@@ -16,7 +16,7 @@ interface Props {
 export default function SIDLogin({ backdropClick }: Props) {
   const epicLoginUrl = 'https://legendary.gl/epiclogin'
 
-  const { epic } = useContext(ContextProvider)
+  const { epicLogin } = useContext(ContextProvider)
```

Comment:
the GlobalState object still exposes the epic login and logout functions since that's not state